zingoconfig/src/lib.rs

```diff
@@ -212,7 +212,7 @@ impl ZingoConfig {
         }
     }
 
-    pub fn get_server_uri(&self) -> http::Uri {
+    pub fn get_lightwalletd_uri(&self) -> http::Uri {
         self.server_uri
```

Comment:
These names should match. `lightwalletd` is a much better name than `server`, let's update the field name.

Reply:
I did this.. but I think it will cause a lot of conflict.